.github/workflows/ci-tests.yml

```diff
@@ -44,12 +44,21 @@ jobs:
           # PyPy3 on Windows doesn't seem to work
           - os: "windows-latest"
             py: "pypy-3.7"
+          # See: https://github.com/actions/setup-python/issues/850
+          - os: macos-latest
+            py: "3.7"
```

Comment:
This version can be removed. We won't support Python 3.7 when the next version of Deform is released. Also remove line 27, as that becomes duplicitous to these workarounds.

Reply:
> Also remove line 27, as that becomes duplicitous to these workarounds.

Do you mean this line:

```diff
        os:
          - "ubuntu-latest"
          - "windows-latest"
-          - "macos-latest"
```

Reply:
Correct.